compose.yml

```diff
@@ -0,0 +1,79 @@
+version: '3' 
```

Comment:
let's rename this file `docker-compose.yml`? I think that's just as standard, and it's more perspicuous

Reply:
done